src/ddmd/attrib.d

```diff
@@ -806,6 +812,21 @@ extern (C++) final class PragmaDeclaration : AttribDeclaration
             }
             return createNewScope(sc, sc.stc, sc.linkage, sc.cppmangle, sc.protection, sc.explicitProtection, sc.aligndecl, inlining);
         }
+        else if (IN_LLVM && ident == Id.LDC_profile_instr) {
```

Comment:
Propose a way to add pragmas to the frontend via a hook.  (Maybe #7457 might be the place to put it).